tools/build_go_exe.sh

```diff
@@ -8,17 +8,16 @@ if [ $# -lt 2 ]; then
     exit 1
 fi
 
-TOOLS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-REPO_ROOT_DIR=$(dirname $TOOLS_DIR)
+source $TOOLS_DIR/init_paths.sh
```

Comment:
The $TOOLS_DIR requirement on calling scripts can be removed by just doing the following:
```
TOOLS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
source $TOOLS_DIR/init_paths.sh
```